bitrisescript/tests/test_script.py

```diff
@@ -35,15 +36,18 @@ async def test_async_main(mocker, config, task, expectation, expected_num_future
     client_mock.close.return_value.set_result(None)
     client_mock.set_app_prefix.return_value.set_result(None)
     mocker.patch("bitrisescript.script.BitriseClient", return_value=client_mock)
+    mocker.patch("bitrisescript.script.get_running_builds", return_value=mocker.AsyncMock())
+    mocker.patch("bitrisescript.script.find_running_build", return_value=mocker.Mock())
+    mocker.patch("bitrisescript.script.wait_and_download_workflow_log", return_value=mocker.AsyncMock())
+    mocker.patch("bitrisescript.script.run_build", return_value=mocker.AsyncMock())
+
+    # Mock signal handler
+    mocker.patch("bitrisescript.script.asyncio.get_event_loop", return_value=mocker.Mock())
```

Comment:
I'm curious, why is this needed now?

Reply:
Because pytest and event loops are a major pain to deal with.
While adding the tests I kept getting an error saying the "event loops is closed". It's usually related to scoping fixtures, but in this case mocking out the signal handler seemed to fix it for me.